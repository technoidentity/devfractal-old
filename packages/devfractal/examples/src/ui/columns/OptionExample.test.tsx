import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Options } from './OptionExample'

test('Options Example of columns', () => {
  const component: ReactTestRenderer = renderer.create(<Options />)
  expect(component.toJSON()).toMatchSnapshot()
})
