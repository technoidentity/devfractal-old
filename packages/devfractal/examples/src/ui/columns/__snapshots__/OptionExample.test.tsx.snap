// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options Example of columns 1`] = `
<div>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Vertical alignment
      </h1>
      <div
        className="columns is-vcentered"
      >
        <div
          className="column is-8"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            vertically centered
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary"
          >
            Second column with more content. This is so you can see the vertical alignment.
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Multiline
      </h1>
      <div
        className="columns is-mobile is-multiline"
      >
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-half is-half"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-half
          </div>
        </div>
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column is-one-quarter is-one-quarter"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            is-one-quarter
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            Auto
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Centering columns
      </h1>
      <div
        className="columns is-mobile is-centered"
      >
        <div
          className="column is-half is-half"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-half
            </code>
          </div>
        </div>
      </div>
      <div
        className="columns is-mobile is-multiline is-centered"
      >
        <div
          className="column is-narrow"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-narrow
            </code>
            <p>
              First Column
            </p>
          </div>
        </div>
        <div
          className="column is-narrow"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-narrow
            </code>
            <p>
              Our Second Column
            </p>
          </div>
        </div>
        <div
          className="column is-narrow"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-narrow
            </code>
            <p>
              Third Column
            </p>
          </div>
        </div>
        <div
          className="column is-narrow"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-narrow
            </code>
            <p>
              The Fourth Column
            </p>
          </div>
        </div>
        <div
          className="column is-narrow"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            <code
              className="html"
            >
              is-narrow
            </code>
            <p>
              Fifth Column
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
